app/models/voice_response.rb

```diff
@@ -0,0 +1,24 @@
+class VoiceResponse < ApplicationRecord
+    
+    def initialize(call)
+        @call = call
+        case @call
+        when BeginCall then @response = Twilio::TwiML::VoiceResponse.new do |r| 
+            r.say(message: @call.message)
+            r.gather(numDigits: @call.numDigits,
+            action: @call.endpoint,
+            method: @call.method)
```

Comment:
Line up your method arguments with each other, or include them all on one line.

Make sure your block code is indented more than your `do` and `end` keywords.